### ISSUE TITLE
[Model] `MacBookPro11,1` wrong

### ISSUE THREAD

**Issue by nuomi1** [RockerHX/Devices#1]
`MacBookPro11,1` is `MacBook Pro (Retina, 13-inch, Late 2013)` **AND** `MacBook Pro (Retina, 13-inch, Mid 2014)`.

https://support.apple.com/en-us/HT201300

**Comment by RockerHX** [comment]
@nuomi1  Yes, I know. Late 2013 and Mid 2014 has same hardware configuration, so you can get screen size to know the specific equipment.

Do you want distinguish "MacBookPro11,1", "MacBookPro11,2" and "MacBookPro11,3" hardware?

**Comment by nuomi1** [comment]
Apple -> About (/Applications/Utilities/System Information.app) can get the real description.

But I don't know how to find the private API (function).

**Comment by RockerHX** [comment]
@nuomi1 Bingo.
